.travis.yml

```diff
@@ -38,6 +39,10 @@ matrix:
         - php: 7.3
           env: SYMFONY_VERSION="^4"
 
+          # Test with Symfony 5
+        - php: 7.3
+          env: SYMFONY_VERSION="^5"
```

Comment:
This is not required because by default the latest Sf 5 version is automatically chosen by composer which already happens above.

Reply:
Yep, i'm not used to travis...
I'll correct this.